tests/test_jartic_plugin.py

```
@@ -0,0 +1,266 @@
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)